<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updateBox</class>
 <widget class="QDialog" name="updateBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>59</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{font-size:30px;}</string>
   </property>
   <property name="text">
    <string>Change User Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{font-size:15px;color:blue;}</string>
   </property>
   <property name="text">
    <string>User name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{font-size:15px;color:blue;}</string>
   </property>
   <property name="text">
    <string>User password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{font-size:15px;color:blue;}</string>
   </property>
   <property name="text">
    <string>Phone number:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{font-size:15px;color:blue;}</string>
   </property>
   <property name="text">
    <string>Address:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>170</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>200</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>240</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>320</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{font-size:15px;color:blue}</string>
   </property>
   <property name="text">
    <string>update</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{font-size:15px;color:blue;}</string>
   </property>
   <property name="text">
    <string>Account number:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
